04-connection_and_https/https/setting_up_https.md

```diff
@@ -167,12 +170,16 @@ http {
 
         # Tell the browser that we always want the user to connect via
         # HTTPS to this site.
-        add_header Strict-Transport-Security "max-age=31536000";
+        add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload";
 
         # Enable OCSP stapling
         ssl_stapling on;
         resolver 8.8.8.8 8.8.4.4 valid=300s;  # DNS servers to reference
         resolver_timeout 5s;
+        
+        # Diffie-Hellman for forward secrecy
+        # It can be generated using `openssl dhparam -out dhparam.pem 4096`
+        ssl_dhparam /etc/ssl/certs/dhparam.pem;
```

Comment:
Nice. Will `openssl` put `dhparam.pem` in `/etc/ssl/certs` by default? If not, the comment should be updated to be copypasta-ble